lib/phoenix_live_view/router.ex

```diff
@@ -46,8 +46,24 @@ defmodule Phoenix.LiveView.Router do
       article_index_path(@socket, :edit, 123)
 
   The current action will always be available inside the LiveView as
-  the `@live_action` assign. `@live_action` will be `nil`
-  if no action is given on the route definition.
+  the `@live_action` assign, that can be used to render a LiveComponent:
+
+      <%= if @live_action == :new do %>
+        <%= live_component @socket, MyAppWeb.ArticleLive.FormComponent %>
+      <% end %>
+
+  Or can be used to show or hide parts of the template:
+
+      <%= if @live_action == :edit do %>
+        <%= render("form.html", user: @user) %>
+      <% end %>
+
+  Note that `@live_action` will be `nil` if no action is given on the route definition.
+
+  In our example, the form component could be displayed on a popup and closing it
+  would be as simple as adding a link to navigate back to the `:index` path:
+
+      <%= live_patch "Close", to: Routes.article_index_path(@socket, :index) %>
```

Comment:
I would remove this section because we would need more context if we want to talk about this. Otherwise, it probably only makes sense if you already know what this is about. :)

```suggestion
```